composer.json

```diff
@@ -80,17 +90,15 @@
   },
   "scripts": {
     "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
+    "create-required-files": "SkilldDrupal\\composer\\ScriptHandler::createRequiredFiles",
+    "list-scaffold-files": [
+      "SkilldDrupal\\composer\\ScriptHandler::listScaffoldFiles"
+    ],
     "pre-install-cmd": [
-      "DrupalProject\\composer\\ScriptHandler::checkComposerVersion"
+      "SkilldDrupal\\composer\\ScriptHandler::checkComposerVersion"
```

Comment:
No need if core scaffolding enabled

Reply:
Also you need this patch to use core's scaffolding https://www.drupal.org/project/drupal/issues/3103090